runtime/Go/antlr/error_strategy.go

```diff
@@ -26,6 +26,8 @@ type DefaultErrorStrategy struct {
 	lastErrorStates   *IntervalSet
 }
 
+var _ ErrorStrategy = &DefaultErrorStrategy{}
```

Comment:
Actually forget it - I see you're asserting this provides an impl of ErrorStrategy!